policy-engine/src/config/settings.rs

```diff
@@ -51,6 +52,17 @@ pub struct AppSettings {
 
     /// Jaeger host and port for tracing support
     pub tracing_endpoint: Option<String>,
+
+    #[default(10)]
+    pub backlog: u32,
+    #[default(10)]
+    pub max_connections: usize,
+    #[default(64)]
+    pub max_connection_rate: usize,
+    #[default(None)]
+    pub keep_alive: Option<Duration>,
+    #[default(Duration::new(5, 0))]
+    pub client_timeout: Duration,
```

Comment:
@vrutkovs Looks like we are setting client_timeout as 5 seconds.  which looks fine to me.  What will be the value for keep_alive?

Reply:
I remember we set the keep_alive somewhere else but can not seem to find it now.

Reply:
Default for PE would be 0. We're setting metrics server keep_alive to [10 seconds](https://github.com/openshift/cincinnati/blob/master/policy-engine/src/main.rs#L132) (so that kubelet requests would re-use the same connection) and [in GB main server](https://github.com/openshift/cincinnati/blob/master/graph-builder/src/main.rs#L131)

Reply:
probably worth documenting where the new defaults are coming from in the commit message, even if the reasoning is just "passing through the current actix default" or "wildly guessing" ;)